pkg/apis/pipeline/v1beta1/resultref_test.go

```diff
@@ -188,6 +207,25 @@ func TestHasResultReference(t *testing.T) {
 			PipelineTask: "sumTask2",
 			Result:       "sumResult2",
 		}},
+	}, {
+		name: "Test valid expression in object",
```

Comment:
just want to make sure I understand what's going on here-- the parameter named "param" is an object param with keys that are each individual results from pipeline tasks?

Reply:
> just want to make sure I understand what's going on here-- the parameter named "param" is an object param with keys that are each individual results from pipeline tasks?

Yes, correct!